### PR TITLE
Settings - Contacts

### PR DESCRIPTION
**Pull request by mayethsc**

### CODE REVIEW

File: application/language/english/hirer/setting_lang.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+$lang['label_contact_viber'] = 'Viber';
+$lang['label_contact_skype'] = 'Skype';
+$lang['label_facebook'] = 'Facebook';
+$lang['label_twitter'] = 'Twitter';
+$lang['label_fax'] = 'Fax';
+$lang['label_website'] = 'Website';
+$lang['label_inquiry_email'] = 'Email Address for Employer Inquiry';
+
+// Contact for applicant inquiries
+$lang['label_applicant_contact_us_form'] = 'Allow applicants to contact our agency via an email form in our Agency Profile Page.';
```
Comment (princeryansy) [inline application/language/english/hirer/setting_lang.php]: please name lang keys with the appropriate prefixes, everything cannot be a label.

File: application/modules/hirer/controllers/Setting.php [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+            dir_hirer($this->currentClass),
+            $this->currentLanguage
+        );
+
+        $this->load->model(model_hirer('Profile'), 'profile');
+    }
+
+    public function contacts()
+    {
+
```
Comment (princeryansy) [inline application/modules/hirer/controllers/Setting.php]: remove blank line

File: application/modules/hirer/views/forms/setting-contacts.php [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+
+// Applicant Inquiries Inputs
+$contactUsFormInput = form_checkbox(
+    array(
+        'name' => 'contact-us-form',
+        'id' => 'contact-us-form',
+        'class' => 'custom-switch-input'
+    ),
+    1,
+    $checked
```
Comment (princeryansy) [inline application/modules/hirer/views/forms/setting-contacts.php]: use set_checkbox method.

File: application/modules/hirer/views/forms/setting-contacts.php [hunk 4]
```diff
@@ -0,0 +1,274 @@
...
+
+$formSubmit = form_button(
+    array(
+        'type' => 'submit',
+        'class' => 'btn btn-primary float-right',
+        'content' => 'Save'
+    )
+);
+
+$hidden = array(
```
Comment (princeryansy) [inline application/modules/hirer/views/forms/setting-contacts.php]: put hidden inputs at the top.

**Comment by mayethsc** [comment]
hi @princeryansy , done updating requested changes.

**Comment by princeryansy** [comment]
ACR Approved!

**Pull request merged**